ca.uwaterloo.Raven.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "ca.uwaterloo.Raven",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "Raven.exe",
+  "finish-args":
+  [
+    "--filesystem=home"
+  ],
+  "cleanup": [
+    "/lib/pkgconfig",
+    "/lib/*.la",
+    "/lib/*.a",
+    "/lib/*.settings",
+    "/lib/debug",
+    "/include",
+    "/share"
+],
+  "modules": 
+  [
+    {
+      "name": "raven",
+      "buildsystem": "simple",
+      "build-commands": [
+        "sed -i 's/^CXXFLAGS += -c++11/#&/' Makefile",
+        "make",
+        "install -D Raven.exe /app/bin/Raven.exe"
```

Comment:
is this a windows executable?

Reply:
no - we have retained the .exe extension so that directions for using the application are uniform across platforms. Not attached to this, however, if it violates rules.